### PR TITLE
WebEngine Projection mode

### PR DESCRIPTION
**Pull request by kshala-ford**
This proposal is created based on discussion in https://github.com/smartdevicelink/sdl_evolution/issues/767.
It extends [SDl-0240 - WebEngine support for SDL JavaScript](https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0240-sdl-js-pwa.md) with a projection mode (formally known as OpenHMI) that was introduce with [SDL-0031 Mobile Projection](https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0031-mobile-projection.md).

### CODE REVIEW

**Comment by kshala-ford** [comment]
@theresalech the PR is ready for review.

File: proposals/NNNN-webengine-projection-mode.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+* Proposal: [SDL-NNNN](NNNN-webengine-projection-mode.md)
+* Author: [Kujtim Shala](https://github.com/kshala-ford)
+* Status: **Awaiting review**
+* Impacted Platforms: [Core / JavaScript ]
+
+## Introduction
+
+This proposal is created based on discussion in https://github.com/smartdevicelink/sdl_evolution/issues/767.
+It extends [SDl-0240 - WebEngine support for SDL JavaScript](https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0240-sdl-js-pwa.md) with a projection mode (formally known as OpenHMI) that was introduce with [SDL-0031 Mobile Projection](https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0031-mobile-projection.md).
```
Comment (theresalech) [inline proposals/NNNN-webengine-projection-mode.md]:
```suggestion
It extends [SDl-0240 - WebEngine support for SDL JavaScript](https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0240-sdl-js-pwa.md) with a projection mode (formerly known as OpenHMI) that was introduced with [SDL-0031 Mobile Projection](https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0031-mobile-projection.md).
```

File: proposals/NNNN-webengine-projection-mode.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+## Motivation
+
+This chapter describes a feature made possible using a WebEngine. A WebEngine can come with a web page rendering component described as a WebView in this proposal. The proposal describes how apps based on a WebEngine can be presented not only using system templates but that can control the WebView using the app's document object.
```
Comment (theresalech) [inline proposals/NNNN-webengine-projection-mode.md]:
```suggestion
This proposal describes a feature made possible using a WebEngine. A WebEngine can come with a web page rendering component described as a WebView in this proposal. The proposal describes how apps based on a WebEngine can be presented not only using system templates but that can control the WebView using the app's document object.
```

File: proposals/NNNN-webengine-projection-mode.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+##### Policy control
+
+The HMI type `PROJECTION` is policy controlled. On the policy server this HMI type can be added to the valid HMI type list per app ID. Only apps with permissions to use this HMI type would be allowed to register.
+
+##### User interface guidelines (Driver Distraction rules)
+
+With the current depth of this proposal, the HMI type should be used by 1st party OEM apps only. With future proposals and workshops the SDLC could open the HMI type to 3rd party by creating and defining proper driver distraction and user interface guidelines.
+
+At the time of this proposal being in review, a set of driver distraction rules are being created and proposed to enable 3rd party using the projection mode. The following bullet points list items that will be described further in the ruleset:
```
Comment (theresalech) [inline proposals/NNNN-webengine-projection-mode.md]:
```suggestion
At the time of this proposal being in review, a set of driver distraction rules are being created and proposed to enable 3rd party using the projection mode. The following bullet points are items that will be described further in the ruleset:
```

File: proposals/NNNN-webengine-projection-mode.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+
+-	minimum font size
+-	minimum contrast between font and background
+-	min/max brightness (for day and night mode)
+-	number of buttons
+-	minimum size of a button
+- No customized keyboard
```
Comment (theresalech) [inline proposals/NNNN-webengine-projection-mode.md]:
```suggestion
- no customized keyboard
```

File: proposals/NNNN-webengine-projection-mode.md [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+- No video playback (exceptions in standstill per countries)
```
Comment (theresalech) [inline proposals/NNNN-webengine-projection-mode.md]:
```suggestion
- no video playback (exceptions in standstill per countries)
```

File: proposals/NNNN-webengine-projection-mode.md [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+-	NHTSA related guidelines
+  o	Amount of text (button and text fields)
+  o	number of lines of text
+  o	Complexity of use cases (number of steps to fulfill a use case)
```
Comment (theresalech) [inline proposals/NNNN-webengine-projection-mode.md]: These last three bullets are not formatting correctly in .md.  Can you please update?

**Comment by kshala-ford** [comment]
@theresalech just made the changes requested.

**Pull request merged**